test/containers/check-bastion

```diff
@@ -25,7 +25,6 @@ import sys
 test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 sys.path.append(test_dir)
 
-from verify import parent
```

Comment:
In general these are needed to set `sys.path` to our test/ and bots/ directories.  So the only thing required from these is this side effect,  no exports. Curiously it seems to work for the avocado test, but the container tests now [fail](https://fedorapeople.org/groups/cockpit/logs/pull-8706-20180226-064242-86d23f53-container-bastion/log.html) due to not finding testvm.py.

The usual pattern to quiesce pyflakes in this case is to just put `parent  # pyflakes` on a line after the import.

Reply:
Alrighty. Fixed.